SPECS/irqbalance/irqbalance.spec

```diff
@@ -33,6 +33,11 @@ make %{?_smp_mflags}
 %install
 make DESTDIR=%{buildroot} install
 install -D -m 0644 misc/irqbalance.env %{buildroot}/etc/sysconfig/irqbalance
+
+# Define IRQBALANCE_ARGS in EnvironmentFile since it is referenced in the service unit file
+# and causes systemd to complain. 
+sed -i 's#\#IRQBALANCE_ARGS=#IRQBALANCE_ARGS=#' %{buildroot}/etc/sysconfig/irqbalance
```

Comment:
Discussed offline but posting here - let's create a patch for this instead of using `sed` to alter the behavior. And we should also share our finding with the newer systemd behavior to the irqbalance upstream community